collector/kernel/bpf_handler.cc

```diff
@@ -139,24 +140,25 @@ void BPFHandler::load_probes(::ebpf_net::ingest::Writer &writer)
   writer.socket_steady_state(0);
 
   // probe for steady-state data
+  LOG::debug("Attempting to instrument tcp rtt estimates.  Several approaches may be tried, depending on the kernel version.");
   int rtt_status = probe_handler_.start_probe(bpf_module_, "on_tcp_rtt_estimator", "tcp_rtt_estimator");
-  if (rtt_status) { // if tcp_rtt_estimator fails
-    log_.warn(
-        "cannot start tcp_rtt_estimator probe, trying "
-        "tcp_update_pacing_rate. error {}",
-        rtt_status);
+  if (rtt_status != 0) { // if tcp_rtt_estimator fails
+    LOG::debug("tcp rtt estimator alternative #1 failed, trying approach #2");
 
     rtt_status = probe_handler_.start_probe(bpf_module_, "on_tcp_rtt_estimator", "tcp_update_pacing_rate");
-    if (rtt_status) {
-      log_.warn("cannot start tcp_update_pacing_rate probe, trying tcp_ack. error {}", rtt_status);
+    if (rtt_status != 0) {
+      LOG::debug("tcp rtt estimator alternative #2 failed, trying approach #3");
 
       rtt_status = probe_handler_.start_probe(bpf_module_, "on_tcp_rtt_estimator", "tcp_ack");
-      if (rtt_status) {
-        log_.warn("cannot start tcp_ack probe. error {}", rtt_status);
-      }
     }
```

Comment:
nit, looks like we could un-nest this? (in a follow up PR if you feel like it, no urgency)